.github/workflows/e2e-tests.yml

```diff
@@ -17,15 +17,19 @@ jobs:
     name: E2E Windows tests
     runs-on: ${{ matrix.os }}
     timeout-minutes: 20
-    env:
-      DEBUG: true
```

Comment:
who knows, this could have been hurting perf too. It looks like this has been here from the very start. 🤷🏼